app/models/webhook_header.rb

```diff
@@ -0,0 +1,22 @@
+# == Schema Information
+#
+# Table name: webhook_headers
+#
+#  id         :integer          not null, primary key
+#  webhook_id :integer
+#  name       :string(255)
+#  value      :string(255)
```

Comment:
Good point. See discussion in the [mailing list](https://groups.google.com/forum/#!topic/portus-dev/x-xuJDRjrIQ).